### DISCUSSION TITLE
21/05/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#164] [Daily Sprint]
Que hice ayer
Que hago hoy
Impedimentos

**Comment by Pamela198713** [comment]
Ayer  tuve una clase. Leí la devolución de la profe de tecnología. Vi las pantallas y seleccione algunas. Hoy instalaré lo necesario haré algunas.
Los tiempos son muy acotados y la organización entre lo laboral y el estudio en el ispc.

**Comment by LuciaL152** [comment]
Ayer estuve refrescando algunas cosas de Android que no me acordaba y aparte dividimos algunas pantallas, hoy estuve practicando hacer algunas cositas en la app

**Comment by fmdcba** [comment]
Hoy estuve modificando el PR para que el administrador no pueda loguearse voy a estar atento a lo que falte y trataré de ir haciendo las correcciones para tecnología y desarrollo

**Comment by rpgrca** [comment]
Hoy seguí avanzando con el carrito, ya está mostrando datos. No tuve clases de emprendedurismo, y estuve en la oficina virtual para ver el problema de práctica profesionalizante que por suerte se solucionó.
